podman/api/client.py

```diff
@@ -7,7 +7,7 @@
 import requests
 from requests.adapters import HTTPAdapter
 
-from podman import api
+from podman import api  # pylint: disable=cyclic-import
```

Comment:
@jwhonce this fixes the cyclic import error we were getting. I think this should be fine as we are only using `api` to get the version number so I don't think having a cyclic import here should break anything. But not entirely sure.

Reply:
Ack, shouldn't be a problem...